content/dstu2/related-person.md

```diff
@@ -31,18 +31,18 @@ Notes:
 
 ### Response
 
-<%= headers 200, GET: '[...]/RelatedPerson?identifier=urn:oid:2.16.840.1.113883.3.13.6|URN:CERNER:IDENTITY-FEDERATION:REALM:AC193C1A-9763-45E1-9FA2-C1C3AA4BA16D:PRINCIPAL:PI98N2FK5TN' %>
+<%= headers 200, GET: '[...]/RelatedPerson?_id=5796348' %>
```

Comment:
I did intentionally use the identifier search it was the more interesting search and I don't really see much benefit in giving 2 examples basically showing the same thing. So my preference would be for it to be searching by identifier, but it isn't _wrong_ to use _id.

If we don't want to use identifier for whatever reason then I would prefer patient over _id